fs/io_zenfs.cc

```diff
@@ -55,22 +55,19 @@ void ZoneExtent::EncodeJson(std::ostream& json_stream) {
 
 enum ZoneFileTag : uint32_t {
   kFileID = 1,
-  kFileName = 2,
-  kFileSize = 3,
-  kWriteLifeTimeHint = 4,
-  kExtent = 5,
-  kModificationTime = 6,
-  kActiveExtentStart = 7,
-  kIsSparse = 8,
+  kFileSize = 2,
```

Comment:
Keep kFilename(just mark is as deprecated) and don't change the other constants, we might want to be able to convert beetween on-disk-formats

Reply:
> Keep kFilename(just mark is as deprecated) and don't change the other constants, we might want to be able to convert beetween on-disk-formats

This will be in 2.0 that changes on disk format anyway?